Sources/Fuzzilli/Lifting/ScriptWriter.swift

```diff
@@ -14,18 +14,51 @@
 
 struct ScriptWriter {
     /// How many spaces to use per indention level.
-    public static let indent = 4
+    public let indent: Int
     
     /// The current script code.
     var code = ""
     
     /// The current number of spaces to use for indention.
-    private var indention: Int = 0
+    private var currentIndention: Int = 0
+
+    private let minifyOutput: Bool
+
+    public init (minifyOutput: Bool, indent: Int = 4) {
+        self.minifyOutput = minifyOutput
+        self.indent = minifyOutput ? 0 : indent
+    }
+
+    /// If minify mode is turned on, remove whitespaces around characters
+    /// which cannot occur in JS variable name
+    func formatLine<S: StringProtocol>(_ line: S) -> String {
```

Comment:
Can you avoid the additional copy in here by directly appending to self.code (including the leading whitespace in case of !minifyOutput) instead of a temporary string instance?

Reply:
I'm not huge fan of functions not returning what they do, but if we optimize additional copying then sure